code/modules/fields/timestop.dm

```diff
@@ -27,6 +27,9 @@
 	for(var/mob/living/L in GLOB.player_list)
 		if(locate(/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop) in L.mind.spell_list) //People who can stop time are immune to its effects
 			immune[L] = TRUE
+	for(var/mob/living/simple_animal/hostile/guardian/G in GLOB.parasites)

```

Comment:
is it not possible to do

 `for(var/mob/living/simple_animal/hostile/guardian/G in L.(soullinkedmobs var, I forgot what it's actually called)`
`>>>>immune[G] = TRUE`

without the need of double locating 🤔

Reply:
Are guardians soullinked? I'll hunt down this supposed var in a moment.

Reply:
They're the reason soul-linking exists iirc